Src/java/engine-jackson/src/test/java/org/opencds/cqf/cql/engine/execution/TranslatorHelper.java

```diff
@@ -50,4 +53,21 @@ public Library translate(String file) throws UcumException, IOException {
 
         return new JsonCqlLibraryReader().read(new StringReader(json));
     }
+
+    public Library readJson(String json) throws IOException {
```

Comment:
1. Not a big one, but should we follow the hapi-fhir standards here too, if so can we rename the input parameter.
2. Can we also rename the method name to reflect its job. It is reading an input string to return a Library. Maybe readForLibrary, getLibrary, or readLibraryFromInputJson.